### PR TITLE
Fix documentation for $selected argument for radio and checkbox dialogs

### PR DESCRIPTION
**Pull request by bertvandepoel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -183,10 +183,12 @@ $builder->info($text, $title = null);
 ```php
 $builder->listCheck(array $list, $text = null, $selected = null);
```
Comment (clue) [inline README.md]: `$selected` should be an array here.

Can you amend your change to explain why? Thanks!

Comment (bertvandepoel) [inline README.md]: Do you prefer a new commit or an extra one?

Comment (clue) [inline README.md]: I think amending your previous commit makes most sense.

Comment (bertvandepoel) [inline README.md]: I made the amend ;-)

Comment (clue) [inline README.md]: Are you sure you've pushed this? :-)

Comment (bertvandepoel) [inline README.md]: Woops, forgot to git add before git commit --amend xD

**Comment by clue** [comment]
Thanks, much appreciated! :shipit:

**Pull request merged**